R/utils.R

```diff
@@ -11,3 +11,7 @@ is_ci <- function() {
 on_windows <- function() {
   (tolower(Sys.info()[["sysname"]]) == "windows")
 }
+
+release_bullets <- function() {
+  c("Update vignette cf https://ropensci.org/blog/2019/12/08/precompute-vignettes/")
```

Comment:
this is a reminder to update the pre-built vignette that will appear when using `usethis::use_release_issue()` for CRAN releases see https://usethis.r-lib.org/reference/use_release_issue.html

Reply:
- I didn't know that one could pre-compile the vignette, that would be great!
- Then the total area of the extracted catchment (in m^2^) is equal to `drainageArea` as computed in the last line of the example. We could simply display that value.
- I'm fine with not including the full example and tip on README. I personally think that having them in the vignette is more important

Reply:
Sorry, I just realized that that the last code lines of the example are missing. I'll send you an updated vignette.